Tests/SourceKitDTests/XCTestManifests.swift

```diff
@@ -23,6 +35,7 @@ extension SourceKitDTests {
 
 public func __allTests() -> [XCTestCaseEntry] {
     return [
+        testCase(CrashRecoveryTests.__allTests__CrashRecoveryTests),
```

Comment:
Oops!  Hope these are working on Linux...

Reply:
Hmm, CI is failing but those errors link to 

```
/home/buildnode/jenkins/workspace/swift-sourcekit-lsp-PR-Linux/tmp/TemporaryDirectory.YD3VQ4/pkg/Package.swift:3:5: error: type annotation missing in pattern
14:18:28 let pack
```

That seems like an unrelated issue that CI is picking up since it was present and enabled before in `SwiftPMWorkspaceTests.testUnparsablePackage` or perhaps something related to the diagnostics changed?

Reply:
It's failing in CrashRecoveryTests.testSourcekitdCrashRecovery, which makes some sense since sourcekitd is in-process on Linux, so that test doesn't really make sense.  We should just skip it on non-Darwin platforms.  Feel free to either do that in this patch (some kind of `try XCTSkipIf(...)`) or else remove the changes related to enabling these tests on Linux and let me or @ahoppen deal with it.

Reply:
Gotcha, done